### PR TITLE
Feature/case filtering indication

### PR DESCRIPTION
**Pull request by nicholasbus**
## Description

Added indication above cases table(s) to show users how the cases are being filtered.  This change adds Ant D Tags and a few new state variables within the code.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have removed unnecessary comments/console logs from my code
- [X] I have made corresponding changes to the documentation if necessary (optional)
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/Cases/CaseTable.js [hunk 1]
```diff
@@ -164,13 +164,76 @@ export default function CaseTable(props) {
...
+  const [currentKey, setCurrentKey] = useState(1);
+
+  useEffect(() => {
+    // TODO: don't know what to put into this useEffect - current use is to keep the filter state in sync.
+    console.log(initialFilters);
+  }, [initialFilters]);
+
+  useEffect(() => {
+    // TODO: don't know what to put into this useEffect - current use is to keep the filter state in sync.
+    console.log(appFilters);
```
Comment (Randyo28) [inline src/components/pages/Cases/CaseTable.js:182]: Nice work! is there a way to have both appFilters and initialFilters in one useEffect?

File: src/components/pages/Cases/CaseTable.js [hunk 2]
```diff
@@ -164,13 +164,76 @@ export default function CaseTable(props) {
...
+  }, [appFilters]);
+
+  // returns processed array of filters
+  const processFilters = filters => {
```
Comment (Randyo28) [inline src/components/pages/Cases/CaseTable.js:186]: I can see this looking clean with a switch statement

**Review by Randyo28 — APPROVED** [review]

File: src/components/pages/Cases/CaseTable.js (hunk 1 above)
Comment (nicholasbus) [reply src/components/pages/Cases/CaseTable.js:182]: This is something that I didn't think of! I'll have to revisit this.

File: src/components/pages/Cases/CaseTable.js (hunk 2 above)
Comment (nicholasbus) [reply src/components/pages/Cases/CaseTable.js:186]: Hmm.. I think a switch statement in this case would convolute things even more. Looping through an object with a lot of keys, many of which are null keys, is the best way I could thing to implement this ```processFilters``` function. I'll take it into consideration though.

**Review by Poonchy — APPROVED** [review]
Changes are good. Work locally.

**Pull request merged**